pages.id/osx/xattr.md

```diff
@@ -0,0 +1,24 @@
+# xattr
+
+> Utilitas untuk bekerja dengan atribut sistem berkas yang diperluas.
+> Informasi lebih lanjut: <https://keith.github.io/xcode-man-pages/xattr.1.html>.
+
+- Daftar atribut yang diperluas kunci:nilai untuk berkas tertentu:
+
+`xattr -l {{berkas}}`
+
+- Menulis atribut untuk berkas tertentu:
```

Comment:
```suggestion
- Tulis atribut untuk suatu berkas:
```

* Use imperative mood for command examples. (see #8576)

* And though not formally specified in guidelines, "suatu" (*"a specified"*) is more preferred than "tertentu" (*"as specified"*) to avoid ambiguity.